webpack.config.babel.js

```diff
@@ -0,0 +1,48 @@
+import path from 'path';
+
+export default {
+  mode: 'production',
```

Comment:
Should be supplied by Gulp (assume it wouldn't get minified if `development`?).

Reply:
👍 See also [this code comment](https://github.com/libero/pattern-library/pull/77/files#diff-ba210a9157252cc983268fdc3aa3ed52R23).